src/test/java/com/code_intelligence/jazzer/mutation/support/TypeSupportTest.java

```diff
@@ -28,13 +28,7 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.params.provider.Arguments.arguments;
 
-import com.code_intelligence.jazzer.mutation.annotation.DoubleInRange;
-import com.code_intelligence.jazzer.mutation.annotation.FloatInRange;
-import com.code_intelligence.jazzer.mutation.annotation.InRange;
-import com.code_intelligence.jazzer.mutation.annotation.NotNull;
-import com.code_intelligence.jazzer.mutation.annotation.WithLength;
-import com.code_intelligence.jazzer.mutation.annotation.WithSize;
-import com.code_intelligence.jazzer.mutation.annotation.WithUtf8Length;
+import com.code_intelligence.jazzer.mutation.annotation.*;
```

Comment:
This looks like the default editor setting that transforms >N imports from the same class into star-imports. I think explicit imports improve clarity. This is done in all of Jazzer code so far.